dts/bindings/sensor/nordic,nrf-temp.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2018, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Nordic nRF Family TEMP node
+version: 0.1
+
+description: >
+    This is a representation of the Nordic nRF TEMP node
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "nordic,nrf-temp"
+
+    reg:
+      type: array
+      description: mmio register space
+      generation: define
+      category: required
```

Comment:
This is just a detail but since this is a new file, I think it would be good to make it look nicer and place this key right after "type", to keep consistency with other properties. Also the empty line before `...` seems to be unneeded.

Reply:
Agreed. Moved up. 
Some files have an empty line before the ..., some do not, and some don't have these dots either (what's their purpose?). But i removed the empty line for this PR.

Reply:
AFAICT these periods are the optional end of a yaml document within a stream. We probably don't need them at all (as well as the opening `---`) in DTS bindings. But they are present in most of these files. I'm not an expert in this field.